packages/lib/gpcircuits/src/circuitParameters.json

```diff
@@ -10,7 +10,7 @@
       "maxTuples": 0,
       "tupleArity": 0
     },
-    6607
+    6608
```

Comment:
I don't always like the idea of generated files being committed to the repo.
In this case, though, it's convenient that this file shows up in the review and lets me see precisely the number of constraints added by a new feature (not many in this case).